### DISCUSSION TITLE
Bootstrap modal

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#6516] [Q&A]
How to use bootstrap modal or statamic modal itself ? I am trying but no way and when i try to include bootstrap js and css the page style be changed. Please any way and help. Thanks

**Comment by martinoak** [comment]
You also need to have jQuery imported. You can simply use CDN such as:
```
<script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
 ```

**Reply by HazemMohamed252** [reply]
When i try to include it the page crashes or no functionality

**Comment by jasonvarga** [comment]
Statamic has a modals feature built in. You should use that instead of adding a new thing.

https://statamic.dev/extending/modals

**Reply by HazemMohamed252** [reply]
I want it for cp not the front. The way statamic uses not help me for the cp

**Reply by jasonvarga** [reply]
What I linked to is for the CP.
